### ISSUE TITLE
Elevated circulating pristanic acid concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9736]
**Preferred term label:**
Elevated circulating pristanic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Concentration of pristanic acid in the blood circulation above the upper limit of normal. PMID:16685654

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

613724
PMID:16685654  1/1
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
